src/main/java/net/jodah/failsafe/proxy/FailsafeInvocationHandler.java

```diff
@@ -0,0 +1,68 @@
+package net.jodah.failsafe.proxy;
+
+import net.jodah.failsafe.CircuitBreaker;
+import net.jodah.failsafe.FailsafeException;
+import net.jodah.failsafe.RetryPolicy;
+import net.jodah.failsafe.SyncFailsafe;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+
+/** Proxy class that retries operations and has a circuit breaker. */
```

Comment:
How about: _Proxy class that retries method calls to the underlying instance under the policies and settings configured for the given {@link Failsafe}._

Reply:
Break into multiple lines?
```java
/**
 * Proxy class that...
 */
```